toolkit/tools/imagegen/attendedinstaller/primitives/enumfield/enumfield.go

```diff
@@ -0,0 +1,252 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
If this is a modified primitive from tview, we need to credit that somewhere here (in a similar fashion to our `customshortcutlist` primitive)

Reply:
To be honest, I don't think there's enough to attribute tview. I used a lot of different examples to understand the code (including navigationbar which inspired the enumfield most heavily). I looked at tview's primitives and couldn't find one that would resemble my code; I think it's diverged too much.

Reply:
Sure, I'll leave that determination up to you